mofacts/client/lib/sessionUtils.js

```diff
@@ -59,6 +59,10 @@ function sessionCleanUp() {
 
   Session.set('schedule', undefined);
 
+  Session.set('isReport', false);
+  Session.set('hiddenItems', []);
+  Session.set('numVisibleCards', 0);
```

Comment:
this is good but we still shouldn't rely on operational ordering to happen to initialize this by side-effect.  can you set this in card.js as well? outside of functions near the top, as putting it in rendered is after we would need it to be initialized